src/localization/modalStrings.js

```diff
@@ -7,6 +7,10 @@ import LocalizedStrings from 'react-native-localization';
 
 /* eslint-disable max-len */
 
+/**
+ * TODO: select_a_reason
+ */
```

Comment:
We usually add this to each language object as a comment, see elsewhere in this file